pkg/disk/utils.go

```diff
@@ -458,36 +459,64 @@ func parseTags(params map[string]string) (map[string]string, error) {
 	return seenTags, nil
 }
 
+var ErrUnknownZone = errors.New("unknown zone")
+
+func getZone(req *csi.CreateVolumeRequest) (string, error) {
+	volOptions := req.GetParameters()
+	zoneID, ok := volOptions[ZoneID]
```

Comment:
I think we should add logic here, the ZoneID parameter should be invalid when pvc uses wffc, at least a warning event if we don't want to make a break change

Reply:
Just tested, if scheduler selects zone A, but we really provision a disk in zone B, scheduler can still assign the Pod to a Node in zone B, and the pod can starts normally.

So yes, if we want to comply with CSI spec, there will be a breaking change here. For users who use WaitForFirstConsumer and zoneID simultaneously, they may get the error now:
```
failed to provision volume with StorageClass "with-zone": rpc error: code = InvalidArgument desc = Invalid parameters from input: disk-e7e63512-0430-4222-8e25-5a7614fb9b26, with error: conflicting zone, parameters specified: cn-beijing-i, accessibility requires: [cn-beijing-g]
```